src/Installer/redist-installer/targets/BundledManifests.targets

```
@@ -31,12 +31,14 @@
       <RestoredNupkgContentPath>$(NuGetPackageRoot)$([MSBuild]::ValueOrDefault('%(NupkgId)', '').ToLower())/$([MSBuild]::ValueOrDefault('%(Version)', '').ToLower())</RestoredNupkgContentPath>
       <MsiNupkgId>%(Identity).Manifest-%(FeatureBand).Msi.$(MsiArchitectureForWorkloadManifests)</MsiNupkgId>
       <RestoredMsiNupkgContentPath>$(NuGetPackageRoot)$([MSBuild]::ValueOrDefault('%(MsiNupkgId)', '').ToLower())/$([MSBuild]::ValueOrDefault('%(Version)', '').ToLower())</RestoredMsiNupkgContentPath>
+      <IncludedInSource Condition="'%(IncludedInSource)' == ''">false</IncludedInSource>
+      <PackageDownload Condition="'$(DotNetBuildSourceOnly)' != 'true' or '$(IncludedInSource)' != 'true'">true</PackageDownload>
```

This doesn't look right.  PackageDownload is true by looking at the binlog.  
```suggestion
      <PackageDownload Condition="'$(DotNetBuildSourceOnly)' != 'true' or '%(IncludedInSource)' != 'true'">true</PackageDownload>
```

---

Yes, my mistake here, thanks.